xml/abstract_commands.xml

```diff
@@ -75,11 +75,17 @@
         </field>
         <field name="0" bits="23" />
         <field name="aarsize" bits="22:20">
-            2: Access the lowest 32 bits of the register.
+            <value v="2" name="32">
```

Comment:
That shows up as:
> 2 (32): Access the lowest 32 bits of the register.

It seems at least slightly awkward for value 2 to have a name of 32 which is just another number.  Then you end up with two numbers before the colon which is possibly confusing.  Maybe "32b" or something like that?

Reply:
Hmm... that is awkward. I explicitly did not choose 32b because in a C macro all letters are upper case, and then is b bits or bytes? I could call it 32bit? That feels unnecessarily verbose, but that's better than confusing.

Reply:
32bit works for me.

Reply:
I don't see an update here.  I thought you were going with 32bit.

Reply:
Huh. For some reason `ag` didn't search that file, but did search other XML files. Fixed.